pom.xml

```diff
@@ -1342,6 +1342,37 @@
         <module>jmeter</module>
       </modules>
     </profile>
+    <profile>
+      <id>macOS</id>
+      <activation>
+        <os>
+          <family>mac</family>
+          <arch>x86_64</arch>
+        </os>
+      </activation>
+      <properties>
+          <es.version.sha512>1e6a707190c274f76f0ebe05400c39a2b923cc1a261ab2e97b3ffe825ad6a73282a31d98284336e4b4f633cc400af47ac8dfad814d636abd497589c8853ec951</es.version.sha512>
+          <es.platform>darwin-x86_64</es.platform>
+          <kb.executable>kibana.sh</kb.executable>
+          <kb.platform>darwin-x86</kb.platform>
+          <kb.installer.extension>tar.gz</kb.installer.extension>
+          <kb.version.sha512>d5eddedc0a945fd085442241fde1652811cd020de62cee630d91e6f6f69089b8c70bcee869dad551beb14a24fa6aee3ac87d1c0e1e0c3d4abd83a65bdcc69823</kb.version.sha512>
+      </properties>
+    </profile>
+     <profile>
+       <id>windows</id>
+       <activation>
+         <os>
+           <family>Windows</family>
+         </os>
+       </activation>
+       <properties>
+         <kb.executable>kibana.bat</kb.executable>
```

Comment:
Should not be defined also the ElasticSearch properties?

Reply:
Sorry I do not understand, these changes were done with you on a local branch back in Feburary and I am just capturing them now for the community.

Reply:
I have added the properties as requested, but am not in a position to test on windows.

(windows is not the subject of this PR)